pkgs/applications/networking/cisco-packet-tracer/8.nix

```diff
@@ -27,16 +27,16 @@
 }:
 
 let
-  version = "8.0.1";
+  version = "8.1.0";
 
   ptFiles = stdenv.mkDerivation {
     name = "PacketTracer8Drv";
     inherit version;
 
     dontUnpack = true;
     src = requireFile {
-      name = "CiscoPacketTracer_${builtins.replaceStrings ["."] [""] version}_Ubuntu_64bit.deb";
-      sha256 = "77a25351b016faed7c78959819c16c7013caa89c6b1872cb888cd96edd259140";
+      name = "Cisco_Packet_Tracer_${builtins.replaceStrings ["."] [""] version}_Ubuntu_64bit.deb";
```

Comment:
The original filename is `CiscoPacketTracer_810_Ubuntu_64bit.deb`

Just importing with a different name seems doesn't work here. I've had to rename the file to make nix accept it and proceed to build

Reply:
I got `Cisco_Packet_Tracer_810_Ubuntu_64bit_2eb10bc3f0.deb` as original filename when downloading, making things even worse. I didn't really had an Idea on how to handle that, so I decided to just strip the hash.

Next best alternative might be to have it as an overridable option, maybe? That makes it easier to customize, and this package is nonetheless never built on hydra.

Reply:
It's already overridable by using overrideAttrs